src/utils/Renderers.tsx

```diff
@@ -361,6 +365,20 @@ export const createWaypointsTableBody = (
   );
 };
 
+
+
+export const tableRowCounterTwo = () =>{
+  return(
+    <div>{rowCounter}</div>
+  )
+}
+
+export const waypointAmount = () =>{
+  return(
+    <div>{waypointCounter}</div>
+  )
+}
```

Comment:
Try derive `rowCounter` and `waypointAmount` from stateManager like what we did here: https://github.com/cBioPortal/installation-map/blob/e957474682f11f3c9cc9042ccc1c1def4637d4eb/src/utils/Renderers.tsx#L350-L366